Maskbook/Extension/MaskWalletCore/Api_JWKResp.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Api_JWKResp.swift
+//  Maskbook
+//
+//  Created by xiaojian sun on 2022/4/24.
+//  Copyright © 2022 dimension. All rights reserved.
+//
+
+import Foundation
+import MaskWalletCore
+import SwiftyJSON
+
+extension Api_JWKResp {
+    var jsonString: String? {
+        var json = JSON()
+        json["crv"] = JSON(rawValue: crv) ?? []
+        json["x"] = JSON(rawValue: x) ?? []
+        json["y"] = JSON(rawValue: y) ?? []
+        json["key_ops"] = JSON(rawValue: keyOps) ?? []
+        json["kty"] = JSON(rawValue: kty) ?? []
+        json["d"] = JSON(rawValue: d) ?? []
+        return json.rawString()
+    }
+}
+
+extension Api_AesJWKResp {
+    var jsonString: String? {
+        var json = JSON()
+        json["alg"] = JSON(rawValue: alg) ?? []
+        json["k"] = JSON(rawValue: k) ?? []
+        json["key_ops"] = JSON(rawValue: keyOps) ?? []
+        json["kty"] = JSON(rawValue: kty) ?? []
+        return json.rawString()
+    }
+}
```

Comment:
Can we just use `Json(resp)` and let `SwiftyJson` check all these nested values? It should be as same as using `Json(rawValue: )` on each property.

Reply:
`    public init(_ object: Any) {
        switch object {
        case let object as Data:
            do {
                try self.init(data: object)
            } catch {
                self.init(jsonObject: NSNull())
            }
        default:
            self.init(jsonObject: object)
        }
    }`
It only support data and types below:
<img width="374" alt="Screen Shot 2022-04-27 at 14 54 58" src="https://user-images.githubusercontent.com/6863518/165459218-31de6fff-dee1-4e81-a773-7406d38543b1.png">